packages/e2e-tests/jest.playwright.config.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+	preset: 'jest-playwright-preset',
```

Comment:
I think we'll lose the failure artifacts feature if we just use the default preset.